### PR TITLE
Init setup

### PR DESCRIPTION
**Pull request by jerry-jma**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+
 const app = express();
 
 // Middleware
 app.use(express.static('dist'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
+// Pls Put all other HTTP request above this GET request &&
+// Pls use endpoints/routes that is different than the Frond End React Router routes
```
Comment (hithisiskhai) [inline server/app.js]: Lines 11 and 12 not needed. Server will be on separate instance as separation of concerns and routes will not interfere with React Router

File: src/Invitation/InvitationForm.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button } from 'react-bootstrap';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export default function InvitationForm() {
+  return (
+    <>
+      <h2>This is InvitaionForm Page</h2>
```
Comment (hithisiskhai) [inline src/Invitation/InvitationForm.js]: Small, but should read 'This is invitation page'

**Review by hithisiskhai — CHANGES_REQUESTED** [review]
Code looks good. For future readability and clean up, unnecessary comments should be addressed. Happy to approve once addressed.

File: src/App.js [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+   * 2. npm run dev:w (if npm run dev is not working)
+   * 3. Add the route path after localhost:3000
+   *
+   */
+  return (
+    <Router>
+      <Routes>
+        <Route path='/' element={<h2>Just localhost</h2>} />
+        <Route path='/sometest' element={<Testing />} />
+        <Route path='/rsvp' element={<RSVP />} />
```
Comment (DylanDKim) [inline src/App.js]: Also small, but could we be a little bit more descriptive with page titles, RSVP could refer to any of the pages on the RSVP section, could be RSVP-gettingStarted or RSVP-preview etc

Comment (jerry-jma) [reply src/App.js]: Hey Dylan, thanks for pointing out. I just made the update. These naming are mainly for testing purpose, all future naming will be more descriptive. 😊

**Review by DylanDKim — APPROVED** [review]

**Review by hithisiskhai — APPROVED** [review]
Looks great. Approved. Thank you!

**Pull request merged**